src/pages/components/progress-bar/usage.mdx

```diff
@@ -0,0 +1,504 @@
+---
+title: Progress bar
+description:
+  A progress bar provides feedback about the duration and progression of a
+  process.
+tabs: ['Usage', 'Style']
+---
+
+<PageDescription>
+
+A progress bar provides feedback about the duration and progression of a
+process.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>Live demo</AnchorLink>
+<AnchorLink>Variants</AnchorLink>
+<AnchorLink>Formatting</AnchorLink>
+<AnchorLink>Content</AnchorLink>
+<AnchorLink>Universal behaviors</AnchorLink>
+<AnchorLink>Modifiers</AnchorLink>
+<AnchorLink>Accessibility</AnchorLink>
+<AnchorLink>Related</AnchorLink>
+<AnchorLink>References</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+
+</AnchorLinks>
+
+## Overview
+
+A progress bar indicates that the user's request has been received and the
+application is making progress toward completing the requested action. Progress
+bars inform users about the status of ongoing processes, the estimated time of
+how long a process will take, or a request is being excuted.
+
+<Row>
+<Column colLg={12}>
+
+![Progress bar hero image](images/progress-bar-usage-1.png)
+
+</Column>
+</Row>
+
+#### When to use
+
+- If a process can take a considerable or unknown amount of time
+- When the process can be described with quantitative information, for example
+  as a percentage
+- To visually show the progression of a system operation such as downloading,
+  uploading, loading data, submitting a form, or saving updates
+- To convey that data is being requested, transferred or processed
+
+#### When not to use
+
+- When the progress bar will be replaced by a minor user interface element once
+  loaded, use [skeleton states](/patterns/loading) instead
+- When manual user actions are required to progress, use a
+  [progress indicator](/components/progress-indicator/usage) instead
+- When the progress is determined by user actions such as tutorials completed,
+  or storage space, rather than system actions
+- If the process takes less than 10 seconds to load, use
```

Comment:
I just noticed some of you gif examples take less than 10 seconds but feel correct. I wonder if 10 seconds is too long?